translations/README.turkish.md

```diff
@@ -0,0 +1,219 @@
+
+## Demolar
+
+Tüm örnek uygulamaları kontrol edin.
+
+[community/sample-apps](community/sample-apps) dizini.
+
+
+### Gerçek zamanlı uygulamalar
+
+- React ile oluşturulan grup shohbet uygulaması, bir yazma belirteci içerir, çevrimiçi kullanıcılar ve yeni mesaj bildirimleri.
+  - [Deneyin](https://realtime-chat.demo.hasura.app/)
+  - [Tutorial](community/sample-apps/realtime-chat)
+  - [APİ'lere göz atın](https://realtime-chat.demo.hasura.app/console)
+
+- Bir harita üzerinde hareket eden mevcut GPS koordinatlarını değiştiren, çalışan bir aracı gösteren canlı konum izleme uygulaması.
+  - [Deneyin](https://realtime-location-tracking.demo.hasura.app/)
+  - [Tutorial](community/sample-apps/realtime-location-tracking)
+  - [APİ'lere göz ayın](https://realtime-location-tracking.demo.hasura.app/console)
+
+- Sürekli değişen veriler üzerinde veri toplama için gerçek zamanlı bir kontrol paneli.
+  - [Deneyin](https://realtime-poll.demo.hasura.app/)
+  - [Tutorial](community/sample-apps/realtime-poll)
+  - [APİ'lere göz atın](https://realtime-poll.demo.hasura.app/console)
+
+### Videolar
+
+* [Kendi kendine barındırılan bir GitLab örneğine GraphQL ekleme](https://www.youtube.com/watch?v=a2AhxKqd82Q) (*3:44 mins*)
+* [Auth0 ve GraphQL backend'iyle todo uygulaması](https://www.youtube.com/watch?v=15ITBYnccgc) (*4:00 mins*)
+* [GitLab auth ile entegre edilmiş GitLab üzerinde GraphQL](https://www.youtube.com/watch?v=m1ChRhRLq7o) (*4:05 mins*)
+* [Coğrafi konumlu 10 milyon sürüş için gösterge tablosu (PostGIS, Timescale)](https://www.youtube.com/watch?v=tsY573yyGWA) (*3:06 mins*)
+
+
+## Destek ve sorun giderme
+Belgeler ve topluluğun çoğu sorunu gidermenize yardımcı olacaktır. Bir hatayla karşılaştıysanız veya bizimle iletişime geçmeniz gerekiyorsa aşağıdaki kanallardan birini kullanarak bizimle iletişime geçebilirsiniz: 
+
+* Destek ve geri bildirim: [Discord](https://discord.gg/vBPpJkS)
+* Sorun ve hata bildirme: [GitHub issues](https://github.com/hasura/graphql-engine/issues)
+* Ürün güncellemelerini takip edin: [@HasuraHQ](https://twitter.com/hasurahq)
+* Bizimle konuşun: [website chat](https://hasura.io)
+
+Tolumda açık ve hoş bir ortam yaratmaya kararlıyız. Lütfen [Code of Conduct](code-of-conduct.md) adresine bakınız
+
+Bir güvenlik sorununu bildirmek istiyorsanız lütfen [bunu okuyun](SECURITY.md).
+
+## Katkıda bulunmak
+
+Daha fazla detay için [katkıda bulunma kuralını](CONTRIBUTING.md) kontrol edin.
+
+## Marka varlıkları
+
+Hasura marka varlıkları (logolar, Hasura maskotu, rozetler vb.) [assets/brand](assets/brand) klasöründedir. Bunları Uygulamanızda/websitenizde kullanmaktan çekinmeyin. Hasura kullanılarak oluşturduğunuz uygulamalarınıza "Powered by Hasura" işareti eklerseniz çok seviniriz. ❤️
+
+<div style="display: flex;">
+  <img src="assets/brand/powered_by_hasura_blue.svg" width="150px"/>
```

Comment:
Logos are not showing here:

![image](https://user-images.githubusercontent.com/5722022/75158486-3c53ca80-573c-11ea-85b9-27e489af1861.png)